### DISCUSSION TITLE
Startup problem orangepi 5 image

### DISCUSSION THREAD

**Discussion by Seb-tech77** [MichaIng/DietPi#7275] [Q&A] [answered]
Hi, I tried to boot on the orange pi 5 image with my orange pi 5 pro but the card doesn’t boot. I tried to make the image with rufus and balenaetcher but nothing to do. I tried to do a bootloader update via the official orangepi image but it didn’t change anything. Could you help me?
Sincerely

**Comment by Joulinar** [comment, marked as answer]
This is now the 4th post for same issue. This doesn't speed up anything.
